tests/integ/modin/frame/test_join.py

```diff
@@ -13,14 +13,16 @@
 
 @pytest.fixture
 def left():
-    return pd.DataFrame(
+    return native_pd.DataFrame(
```

Comment:
as above--how come these are now native pandas objects?

Reply:
I changed these because they greatly increased the query count with the new change to the constructor. It did not seem right to create a Snowpark pandas object and call `to_pandas` to get the native pandas equivalent. I believe that it should be the other way around.